client/lib/query-manager/theme/test/index.js

```diff
@@ -1,17 +1,57 @@
 /**
  * External dependencies
  */
-//import { expect } from 'chai';
+import { expect } from 'chai';
 
 /**
  * Internal dependencies
  */
-//import ThemeQueryManager from '../';
+import ThemeQueryManager from '../';
 
 /**
  * Constants
  */
 
 describe( 'ThemeQueryManager', () => {
-	// TODO
+	let manager;
+
+	beforeEach( () => {
+		manager = new ThemeQueryManager();
+	} );
+
+	describe( '#removeItem()', () => {
+		it( 'should remove an item by its item key', () => {
+			manager = manager.receive( [ { ID: 144 } ] );
```

Comment:
These tests are taken from [`QueryManager`](https://github.com/Automattic/wp-calypso/blob/24f141b6c19ddc699ace7e81f3d08dc02d99e147/client/lib/query-manager/test/index.js#L150) so we may want to keep them the same.